sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -79,23 +82,29 @@
     /**
      * Pending operation CREATE value.
      */
-    private static final String PENDING_OPERATION_CREATE_VALUE = "CREATE";
+    static final String PENDING_OPERATION_CREATE_VALUE = "CREATE";
```

Comment:
Please merge with develop branch, these 3 strings have been moved to Constants.java.